features/buyers/accounts/applications/new.feature

```diff
@@ -26,8 +26,8 @@ Feature: Audience > Accounts > Listing > Account > Applications > New
       | Application plan | Basic          |
       | Name             | New App        |
       | Description      | This is an app |
-    Then the current page is application "New App" admin page
-    And they should see a toast alert with text "Application was successfully created"
+    Then they should see a toast alert with text "Application was successfully created"
```

Comment:
Here and in other places in this file - same as https://github.com/3scale/porta/pull/4111/files#r2216670779